Ginger/GingerCore/Drivers/Mobile/Perfecto/PerfectoDriver.cs

```diff
@@ -473,6 +477,15 @@ private void PressBtn(ePressKey PressKey)
             mDriver.ExecuteScript("mobile:presskey", params1);
         }
 
+        private void TakeScreenShot()
```

Comment:
perfecto changes are not related...
if it's needed pls create PR to master

Reply:
Undone Perfecto changes in latest push.